charts/external-secrets/templates/secrets.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.secretsSource.enabled }}
```

Comment:
do you expect to have secrets in this repo, or secret stores? if secret stores, it could make sense to name the different resources around it ... for example 

`secretsSource` as `secretStores` or 

`name: external-secrets` as `name: secret-stores`

Reply:
No I don't expect to have secrets in this repo, But this option in case the user want to install the profile without defining SecretStore/ExternalSecret location and want to do them later. If enabled=false this will install the operator only

Reply:
ahh okay, then I could think that using secret-store as term instead of secret could help users understanding as there wont be room to understand secrets as the secret data or a kubernetes secret